install.md

```diff
@@ -1,104 +1,52 @@
 ---
 layout: page
 title: Installing Jupyter
-tagline: Get up and running with JupyterLab or the classic Jupyter Notebook.
+tagline: Get up and running on your computer
 permalink: /install
 ---
 
-## Getting started with JupyterLab
+Project Jupyter's tools are available for installation via the [Python Package Index](https://pypi.org/), the leading repository of software created for the Python programming language. Its offerings can be installed with a variety of package management tools that run via a [command-line interface](https://en.wikipedia.org/wiki/Command-line_interface), including [conda](https://docs.conda.io/), [mamba](https://mamba.readthedocs.io/) and [pipenv](https://pipenv.pypa.io/).
 
-_The [installation guide](http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) contains more detailed instructions_
+Which one you use is up to you. Pipenv is recommended by the [Python Packaging Authority](https://www.pypa.io/). Conda and mamba are preferred by many members of [PyData](https://pydata.org/) and can by acquired via [miniforge](https://github.com/conda-forge/miniforge#mambaforge).
 
-### Installation with mamba or conda
+## JupyterLab
 
-JupyterLab can be installed with `mamba` and `conda`:
+Install JupyterLab with one of the following:
 
 ```bash
+pipenv install jupyterlab
```

Comment:
```suggestion
pip install jupyterlab
```

Reply:
Could we leave the installation steps to just `pip`? I think having multiple installation options will confuse people, assuming that most people that install jupyter by following the instructions on this page are novices who are less-experienced with Python in general. The less information we need to give them to get started, the better.